src/main/webapp/app/exam/exam-scores/exam-scores-routing.module.ts

```diff
@@ -0,0 +1,22 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import { UserRouteAccessService } from 'app/core/auth/user-route-access-service';
+import { ExamScoresComponent } from 'app/exam/exam-scores/exam-scores.component';
+
+const routes: Routes = [
+    {
+        path: ':examId/scores',
+        component: ExamScoresComponent,
+        data: {
+            authorities: ['ROLE_ADMIN', 'ROLE_INSTRUCTOR'],
+            pageTitle: 'artemisApp.examScores.title',
+        },
+        canActivate: [UserRouteAccessService],
+    },
+];
+
+@NgModule({
+    imports: [RouterModule.forChild(routes)],
+    exports: [RouterModule],
+})
+export class ArtemisExamScoresRoutingModule {}
```

Comment:
I think this routes wouldn't be lazy loaded. Have a look at `exam-management.route.ts`, especially `examManagementState` routes (also see [Angular's documentation](https://angular.io/guide/lazy-loading-ngmodules)).

I would also suggest to stick to our naming convention --> `exam-scores.route.ts`

Reply:
I've overseen `RouterModule.forChild(routes)`. So lazy loading should be fine but I would still suggest to change the filename.